docs/source/reference/configuration.rst

```diff
@@ -163,6 +163,11 @@ Configuration Keys
    You can use this flag when implementing your own Link to avoid updating the internal states during recomputation done by :func:`chainer.functions.forget`.
    See the documentation of :func:`chainer.functions.forget` for details.
 
+* ``use_static_graph`` (default: ``True``)
+   Flag to configure whether or not to use the static subgraph optimization feature.
+   Where the static subgraph graph optimization decorator is used, we generally assume that that the feature should be used and the default value is thus ``True``.
```

Comment:
* static subgraph graph optimization  -> static subgraph optimization 
* that that -> that

Reply:
Thanks, fixed.